Umbraco-Uno/Creating-Content/How-To-Add-A-New-Font-Style/index.md

```diff
@@ -0,0 +1,36 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to add a new font style through the backoffice
+
+If you find your self in need of a font that is not already included in Umbraco Uno, then you have the option to add one your self. In this article, you will learn how to do that.
+
+What you will have to do is to go to https://fonts.google.com/ select a font you like and click the box.
+
+![The box with the font](images/Click-the-box.png)
+
+After clicking the box, you will be presented with a list of versions of the stile. You can select one of them or all of them. To select them, you click ***+ select this style*** on the style that you want.
+
+![this lets you select style](images/The-select-style.png)
+
+Then a window will slide in from the right, and from here, you select embed. If you do not see the slide in the window, you can press the button in the top right looking like this.
+
+:::Center
+![button to show slide window](images/If-missing-embed-menu.png)
+:::
+
+When you click embed, there will be two boxes. The fist box will be a link you have to copy.
+
+:::Center
+![embed window with link](images/Embed-menu.png)
+:::
+
+Go to your backoffice> settings>theme, then navigate to embed font field and paste the link in there. After this is done, then you select the body font and the heading font.
```

Comment:
Would it make sense to add a bit more details on how to add the new fonts to heading and body?

```suggestion
Go to your Theme settings page in the Umbraco Uno backoffice, navigate to the **Embed Font** field and paste in the link. After this is done, then you can add the body font and the heading font.
```